### PR TITLE
Module refactor

### PR DESCRIPTION
**Pull request by Kledin85**

### CODE REVIEW

File: lib/modules/calculations.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module StandardDeviation
+module Calculations
```
[the remark this answers is not in the document]
Comment (Pocketzs) [reply lib/modules/calculations.rb:1]: Ignore

File: lib/sales_analyst.rb [hunk 2]
```diff
@@ -1,13 +1,14 @@
-require_relative '../lib/modules/standard_deviation'
+require_relative '../lib/modules/calculations'
```
[the remark this answers is not in the document]
Comment (Pocketzs) [reply lib/sales_analyst.rb:1]: Ignore

**Review by beddings81 — APPROVED** [review]
looks good to me!

File: lib/sales_analyst.rb [hunk 3]
```diff
@@ -19,45 +20,48 @@ def average_item_price_std_dev
...
-    (averages.sum / averages.length).round(2)
+  end
+
+  def average_average_price_per_merchant
+    average(merchant_averages)
   end
 
   def prices(merchant_id)
-    @engine.merchants.find_by_id(merchant_id).items.map do |item|
+    merchants.find_by_id(merchant_id).items.map do |item|
```
[the remark this answers is not in the document]
Comment (Pocketzs) [reply lib/sales_analyst.rb:51]: Ignore keep using block syntax

File: lib/sales_analyst.rb [hunk 4]
```diff
@@ -81,59 +85,65 @@ def average_invoices_per_merchant_standard_deviation
...
+
+  def days
+    invoices.all.map do |invoice|
+      invoice.created_at.strftime('%A')
+    end
+  end
+
+  def top_days
+    std_dev = average_invoices_per_day_standard_deviation
+    invoice_by_days.find_all do |day, count|
```
[the remark this answers is not in the document]
Comment (Pocketzs) [reply lib/sales_analyst.rb:114]: Ignore

**Review by KaraJoHo — APPROVED** [review]
Nice refactoring

File: spec/sales_analyst_spec.rb [hunk 5]
```diff
@@ -254,10 +254,10 @@
 
   it 'can calculate percentages by the status' do
     sales_engine = SalesEngine.from_csv(
-      
-      :items     => './data/items_test.csv',
-      :merchants => './data/merchant_invoices_test2.csv',
-      :invoices  => './data/invoices_test2.csv'
+
```
[the remark this answers is not in the document]
Comment (Pocketzs) [reply spec/sales_analyst_spec.rb:257]: Delete line 257

**Review by Pocketzs — CHANGES_REQUESTED** [review]

**Review by Pocketzs — APPROVED** [review]

**Pull request merged**